shared/src/license.ts

```diff
@@ -39,13 +33,54 @@ export type LicenseStatus =
   | "SUSPENDED"
   | "REVOKED";
 
-export interface LicenseData {
+export type LicenseDetails = {
   nameAndAddress: LicenseSearchNameAndAddress;
-  completedSearch: boolean;
-  expirationISO?: string;
+  licenseStatus: LicenseStatus;
+  expirationDateISO: string | undefined;
+  lastUpdatedISO: string;
```

Comment:
dust - if we're doing expirationDateISO, shouldn't the string be lastUpdatedDateISO

Reply:
I kind of like expirationDate more than lastUpdated.. Its my personal opinion but not a hill i will die on

Reply:
I agree with Aaron that we should keep this uniform

Reply:
I like this idea, but FWIW it looks like every other place in our codebase uses `lastUpdatedISO` (see `taxFilingData` for example). I'd rather see us address this as a fast follow everywhere and have a consistent name throughout.